Bottom: 0cf18140759f544d7939aa62d2647161675c47b4
Top:    ec7b8a91970739eed29b1338c43afe9f72f491bc
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-01 16:30:38 +0800

Refresh of autofs-5.0.7-debug.patch

---

diff --git a/modules/lookup_file.c b/modules/lookup_file.c
index 9a5bb8b..b70b344 100644
--- a/modules/lookup_file.c
+++ b/modules/lookup_file.c
@@ -443,7 +443,7 @@ int lookup_read_master(struct master *master, time_t age, void *context)
 
 			inc = check_master_self_include(master, ctxt);
 			if (inc) 
-				master->recurse = 1;;
+				master->recurse = 1;
 			master->depth++;
 			status = lookup_nss_read_master(master, age);
 			if (!status) {
@@ -451,6 +451,11 @@ int lookup_read_master(struct master *master, time_t age, void *context)
 				     MODPREFIX
 				     "failed to read included master map %s",
 				     master->name);
+				/*
+				 * + map include failures don't cause the map
+				 * read to fail
+				 */
+				master->read_fail = 0;
 			}
 			master->depth--;
 			master->recurse = 0;
